src/plugin/memory-webview-main.ts

```diff
@@ -208,6 +213,10 @@ export class MemoryWebview implements vscode.CustomReadonlyEditorProvider {
         return participant;
     }
 
+    protected setTitle(webviewParticipant: WebviewIdMessageParticipant, title: string): void {
```

Comment:
It looks like this is coming from #69. I'd prefer that each PR be independent, if possible.

Reply:
Yes, the PRs have been building on top of each other, with separate commits though, as mentioned in the descriptions. They don't logically depend on each other but change overlapping parts in the code--this is why I originally stacked the PRs.
But I can resolve the conflicts also upon merging. I've detached this commit into an own PR now.